docs/src/forc_client.md

```diff
@@ -1,3 +1,29 @@
 # forc-client
 
 Forc plugin for interacting with a Fuel node.
+
+## Signing transactions using `forc-wallet` CLI
+
+To submit the transactions created by `forc deploy` or `forc run`, you need to sign them first (unless you are using a client without utxo validation). To sign a transaction you can use `forc-wallet` CLI. This section is going to walk you through the whole signing process.
+
+To install `forc-wallet` please refer to `forc-wallet`'s [github repo](https://github.com/FuelLabs/forc-wallet).
+
+1. Construct the transaction by using either `forc deploy` or `forc run`. To do so simply run `forc deploy` or `forc run` with desired parameters. For a list of parameters please refer to [forc-deploy](./forc_deploy) or [forc-run](./forc_run) section of the book. Once you run any of them, you will be asked the address of the wallet you are going to be signing with. After the address is given the transaction will be generated and you will be given a transaction ID. At this point CLI will actively wait for you to insert the signature.
+2. Take the transaction ID generated in the first step and sign it with `forc wallet sign <transaction_id> <account_index>`. This will generate a signature.
+3. Take the signature generated in the second step and provide it to `forc-deploy` (or `forc-run`). Once the signature is provided, the signed transaction will be submitted.
+
+## Initializing the wallet and adding accounts
+
+If you don't have an initialized wallet or any account for your wallet you won't be able to sign transactions.
+
+To initialize a wallet you can use `forc wallet init`. It will ask you to choose a password to encrypt your wallet. After the initialization is done you will have your mnemonic phrase.
+
+After you have an initialized wallet, you can create an account for it by simply running `forc wallet new`. It will ask your password to decrypt the wallet before creating an account.
+
+## Other useful commands of `forc-wallet`
+
+1. You can see a list of your already created accounts by running `forc wallet list`.
```

Comment:
```suggestion
1. You can see a list of existing accounts by running `forc wallet list`.
```

Reply:
Also, you can also possibly format this 'other useful commands' section similar to the Quickstart section, where there's a short description followed by the command wrapped in the `sh` block. That way readers can click to copy the command. This is just a minor suggestion though!